website/project/model.py

```diff
@@ -4241,3 +4250,33 @@ def validate_metadata(self, *args, **kwargs):
         Validates draft's metadata
         """
         return self.registration_schema.validate_metadata(*args, **kwargs)
+
+
+class PreprintProvider(StoredObject):
```

Comment:
Also, should this class have a `url` or `external_url` that points to, e.g. `socarxiv.org`?